src/client.rs

```diff
@@ -73,7 +73,7 @@ impl<'a> Client<'a> {
         name: impl Into<String>,
         value: impl Into<String>,
     ) -> Result<Entry, Error> {
-        let (ctx, _ctx_handle) = Context::new();
+        let (ctx, _ctx_handle) = Context::with_timeout(Duration::from_millis(100));
```

Comment:
How did we decide on the value here?  Can we create a constant for this?

Reply:
Switched it to being a constant. I was originally testing over TCP connection on non-prod version of the app but figured I would try it out over serial on Windows. Apparently it was struggling with such a short timeout, so I bumped this up to a second which seems to work well consistently.